File: AlkmaarseRestaurants/angular/src/app/shared/models/Restaurant/RestaurantMenuItemModel.ts
import { IRestaurantMenuItemModel } from "./IRestaurantMenuItemModel";
import { RestaurantMenuItemCategoryModel } from "./RestaurantMenuItemCategoryModel";

export class RestaurantMenuItemModel implements IRestaurantMenuItemModel {
    public id: number;
    public name: string;
    public description: string;
    public price : number;
    public menuCategory: RestaurantMenuItemCategoryModel;

    constructor() {
        this.id = 0;
        this.name = "";
        this.description = "";
        this.price = 0.00;
        this.menuCategory = new RestaurantMenuItemCategoryModel();
    }
}

File: AlkmaarseRestaurants/Models/Restaurant/RestaurantMenuItemModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AlkmaarseRestaurants.Models.Restaurant
{
    public class RestaurantMenuItemModel
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public RestaurantMenuItemCategoryModel MenuCategory { get; set; }
    }
}
